browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2031,7 +2031,10 @@ Are you sure you want to do this?
       Verified password did not match
     </message>
     <message name="IDS_CREATING_WALLET_ONBOARDING_TEXT" desc="Text displayed along with a spinner during Brave Wallet creation or restoring.">
-      Creating Wallet...
+      Creating Wallet
```

Comment:
In Figma `Wallet` is lowercase, although I am not sure what the right case here should be. Could we be a little more wordy here and say `Creating your wallet` or `Creating Brave Wallet`? cc: @rmcfadden3

Reply:
@mkarolin — at this point since they're pretty deep into the experience, we can probably remove the branding. And if there's no branding, then no capitalization necessary.

I'd vote for `Creating your wallet…`

Reply:
Thank you both! I'm chaing it to `Creating your wallet…` then.

Reply:
Here is a preview. 
I slightly prefer a version without the ellipsis in the title, so it would be `Creating you wallet` as too many ellipsis (title and subtitle) make the UI a bit heavy. 
Just a mild preference though, I'm fine with both.

<img src="https://github.com/brave/brave-core/assets/3308503/8962a6b0-dbce-466c-bc53-689911073de2" width="500" />

Reply:
@simoarpe — sorry for the delayed reply, I can see this one was already merged.

But note we should keep the ellipsis "…" since in this case it's actually a style thing to denote an action that's in progress.

Reply:
In case you're curious, here's the entry for ellipses from the Brave Style Guide:
<img width="730" alt="Screenshot 2024-03-14 at 5 27 37 PM" src="https://github.com/brave/brave-core/assets/78937892/4c5afccb-03d8-4afb-a54c-ee7986dfeba2">